app/src/test/java/com/example/partyPlayer/UnitTest.kt

```diff
@@ -1,15 +1,14 @@
-package com.example.party_player
+package com.example.partyPlayer
```

Comment:
Changed the package names by removing underscores to comply with Kotlin naming conventions stated here: https://kotlinlang.org/docs/reference/coding-conventions.html